test_util/test_upgrade_vpc.py

```diff
@@ -307,7 +307,9 @@ def setup_cluster_workload(self, dcos_api: DcosApiSession, healthcheck_app: dict
 
             dcos_api.marathon.deploy_app(viplisten_app)
             dcos_api.marathon.ensure_deployments_complete()
-            dcos_api.marathon.deploy_app(viptalk_app)
+            # viptalk app depends on VIP from viplisten app, which may still fail
```

Comment:
If ignore any task failure now, and proceed with the upgrade (due to latency), we have to make sure that the task failure is not persistent after the upgrade.  

```
        self.verify_apps_state(dcos_api_upgrade, dns_app)
```

could be modified to include the viplisten and talkapps?